libs/tree-sitter-wing/grammar.js

```diff
@@ -679,6 +702,13 @@ function braced(rule) {
   return choice(seq("{", rule, "}"));
 }
 
+/*
+ * @param {Rule} rule
+ */
+function bracketed(rule) {
+  return choice(seq("[", rule, "]"));
```

Comment:
Does this need to wrapped in a choice()? I'm noticing now that `braced()` also uses a seemingly unnecessary choice(). If you remove the choice from this and braced(), do all the tests still pass?

Reply:
Let me check!

Reply:
There's one test that's flaky since it passed OK in my local with the same set of changes, and it's not even related to the `Set` change.

![image](https://github.com/winglang/wing/assets/13340320/7f13a7d6-1e72-417d-b0d5-e65ee20e2498)

![image](https://github.com/winglang/wing/assets/13340320/8b3a9d7b-47f7-40ee-ad72-e9d8ece47c6d)

Reply:
Indeed it worked